src/main/java/http_server/RequestParser.java

```diff
@@ -2,18 +2,21 @@
 
 public class RequestParser {
 
-    public String parse(String request) {
+    public String parseResource(String request) {
         return request.substring(5, request.lastIndexOf(" "));
     }
 
     public String extractMethodFromRequest(String request) {
         return request.substring(0, request.lastIndexOf(" /"));
     }
 
-    public String parseContentType(String request) {
-        int startOfString = request.indexOf(".", 0) + 1;
-        int endOFString = request.lastIndexOf(" ");
-        return request.substring(startOfString, endOFString);
+    public String parseContentType(String resource) {
+        if (resource.equals("file1") || resource.equals("file2")) {
```

Comment:
This may be too specific to cobspec. file3.txt or whatevs.txt should work as well